### ISSUE TITLE
Takedown notice

### ISSUE THREAD

**Issue by Kaiddd** [HarryBlueJay/RCO2-Patch-Kit#3]
Firstly before I get into my main concern, the section "Why do I have to say this?" is false, I didn't even know this was a repo till just now.

Secondly, this repository is in violation of applicable copyright law in both the United States, and the U.K., along with most other countries in the world that recognize copyright law, as well as the fact that github operates in the United States.

You have 7 days to delete this repository before I file an actual legal takedown notice.

Finally, why do you code for, and support a known pedophile (xhax)?

**Comment by Kaiddd** [comment]
@HarryBlueJay @L8X

Ping

**Comment by HarryBlueJay** [comment]
i decided this would be the perfect time to make a response, you licensed rclientoptimizer2 under gpl 3 which says i can modify and distribute and since rco2 is just an updated version of that, i think i'm legally protected here, and i'm not budging i made a deal with xhax and i do NOT plan on breaking it

**Comment by Kaiddd** [comment]
Hello, thanks for the response, but actually your repo is in violation of every single term under the GPL-3.0 license.

Here are the terms that you need to abide by:

License and copyright notice
State changes
Disclose source
Same license

I will give you till the end of your 7 days to make these changes, thank you.

**Comment by Kaiddd** [comment]
> i don't know legal stuff well but how would i do the first 3

Simply do the following.

Add to readme:
Licensed under GPL-3.0 | Copyright https://github.com/Kaiddd

Add to readme:
A list of changes you've made from my original RCO2 project

Add to readme:
Original source: https://github.com/Kaiddd/roblox-client-optimizer

Add this LICENSE file to the github page
https://github.com/Kaiddd/RobloxClientOptimizer2/blob/main/LICENSE


Afterwards, do the same thing to your RCO3 repo, that is also in violation of GPL-3.0

**Comment by HarryBlueJay** [comment]
this good?
# Copyright Stuff
Licensed under GPL-3.0 | Copyright https://github.com/Kaiddd
List of changes:
The installer menu has been completely redesigned
The installer is much more informative
The installer can update itself
Added a configuration menu "inspired" by RCO3's
Added a help menu "inspired" by RCO3's
Added alternate endpoints for the flag version and flags
Switched the version endpoint and added filtering to it
Switched the program version endpoint
Original source: https://github.com/Kaiddd/roblox-client-optimizer

**Comment by Kaiddd** [comment]
Yes, I'll accept this. No hate towards your project, I have no problem with forks of anything I've coded, so long as they properly follow GPL, I just don't understand the mental gymnastics that make you want to work with a known pedophile, instead of running an independent fork

**Comment by Kaiddd** [comment]
@HarryBlueJay also, at the top, you can't say RCO2 is a part of the optifury project, you must use "RCO2-Patch-Kit" or "This fork of RCO2"

**Comment by Kaiddd** [comment]
Final legal problem, you can't do this in the source

    std::cout << "Credits to an individual who shall remain unnamed for the RCO2 source code. | FFlag list maintained by L8X\n\nRCO is currently: ";

I must get proper attribution, you must also update the latest release to include this new attribution, and delete all previous releases not including this attribution.

**Comment by Kaiddd** [comment]
@HarryBlueJay sorry for ping, issue still open

**Comment by HarryBlueJay** [comment]
will fix in about an hour

**Comment by HarryBlueJay** [comment]
readme has been fixed and so has the executable

**Comment by Kaiddd** [comment]
Does the latest release executable have proper attribution? Will check when home.
